docker-compose.dataservice.yml

```diff
@@ -15,6 +15,8 @@ services:
       - PG_HOST=dataservice-pg
     depends_on:
       - dataservice-pg
+    ports:
```

Comment:
I don't think this needs to change in this PR?

Reply:
Sorry, some extra files got added in. Removed